mls.json.bak

```diff
@@ -0,0 +1,162 @@
+            ],
+            "topic": "TLS Working Group Drafts"
+        }
+    },
+    "cellar@ietf.org": {
+        "digest:sunday": {
+            "eventFilter": {
+                "notlabel": [
+                    "editorial"
+                ]
+            },
+            "repos": [
+                "ietf-cellar-wg/ebml-specification",
+                "ietf-cellar-wg/matroska-specification",
+				"FFmpeg/FFV1",
+				"ietf-cellar-wg/Cellar-FLAC",
+				"ietf-cellar-wg/chair-notes"
+            ],
+            "topic": "CELLAR Activity Summary"
+        }
+    },
+    "mls@ietf.org": {
+        "digest:sunday": {
+            "repos": [
+                "mlswg/mls-architecture",
+                "mlswg/mls-protocol",
+                "mlswg/mls-federation"
+            ],
+            "topic": "MLS Working Group summary"
+        }
+    },
+    "masque@ietf.org": {
+        "digest:sunday": {
+            "repos": [
+                "ietf-wg-masque/draft-ietf-masque-connect-udp",
+                "ietf-wg-masque/draft-ietf-masque-ip-proxy-reqs"
+            ],
+            "topic": "MASQUE Working Group summary"
+        }
+    },
+    "privacy-pass@ietf.org": {
+        "digest:sunday": {
+            "repos": [
+                "ietf-wg-privacypass/base-drafts"
+            ],
+            "topic": "Privacy Pass Working Group summary"
+        }
+    },
+    "mops@ietf.org": {
+        "digest:sunday": {
+            "eventFilter": {
+                "notlabel": [
+                    "editorial"
+                ]
+            },
+            "repos": [
+                "ietf-wg-mops/draft-ietf-mops-streaming-opcons"
+            ],
+            "topic": "MOPS Activity Summary"
+        }
+    },
+    "add@ietf.org": {
+        "digest:sunday": {
+            "eventFilter": {
+                "notlabel": [
+                    "editorial"
+                ]
+            },
+            "repos": [
+                "ietf-wg-add/draft-add-requirements",
+                "ietf-wg-add/wg-materials"
+	    ],
+            "topic": "ADD Activity Summary"
+        }
+    },
+    "edm@iab.org": {
+        "digest:sunday": {
+            "repos": [
+                "intarchboard/edm"
+            ],
+            "topic": "EDM Activity Summary"
+        }
+    },
+    "asdf@ietf.org": {
+        "digest:sunday": {
+            "eventFilter": {
+                "notlabel": [
+                    "editorial"
+                ]
+            },
+            "repos": [
+                "ietf-wg-asdf/SDF"
+            ],
+            "topic": "ASDF Activity Summary"
+        }
+    },
+    "txauth@ietf.org": {
+        "digest:sunday": {
+            "eventFilter": {
+                "notlabel": [
+                    "editorial"
+                ]
+            },
+            "repos": [
+                "ietf-wg-gnap/core-protocol"
+            ],
+            "topic": "GNAP Weekly GitHub Activity Summary"
+        }
+    }
+}
```

Comment:
Could you please remove this from the commit?

Reply:
Thanks for spotting this. I have Notepad++ doing automatic backups, and I .gitignore those in my repos. Just not in yours!